packages/react-tooltip/src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -126,7 +126,6 @@ describe('Tooltip', () => {
     );
 
     const target = result.getByRole('button');
-    expect(target.getAttribute('aria-label')).toBe(null);
```

Comment:
This assertion is not related to this test ("doesn't override trigger's aria-labelledby").

It's expected that we could be in such state as we can't rely on reading props. We can add a runtime warning, for example.

Reply:
Reverted.